### PR TITLE
PS-5938: Post push fix, mismerge is sql_table.cc

### PR DESCRIPTION
**Pull request by dutow**

### CODE REVIEW

File: mysql-test/data_upgrade [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/dutow/workspace/topics/m8019/80-local/mysql-test/var/1/mysqld.1/data/
```
Comment (percona-ysorokin) [inline mysql-test/data_upgrade]: This is a temp file - should not be a part of the commit

Comment (dutow) [reply mysql-test/data_upgrade]: Fixed.

**Comment by dutow** [comment]
https://ps80.cd.percona.com/job/percona-server-8.0-param/536/

**Comment by dutow** [comment]
https://ps80.cd.percona.com/job/percona-server-8.0-param/537/

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**